lms/envs/common.py

```diff
@@ -2088,3 +2088,19 @@
         'profile_image',
     ],
 }
+
+# PROFILE IMAGE CONFIG
+# TODO: add these settings to aws.py as well
+PROFILE_IMAGE_BACKEND = 'django.core.files.storage.FileSystemStorage'
+# PROFILE_IMAGE_DOMAIN points to the domain from which we serve image
+# files from.  When this is '/', it refers to the same domain as the
+# app server.  If serving from a different domain, specify that here
+# i.e. 'http://www.example-image-server.com/'
+PROFILE_IMAGE_DOMAIN = '/'
+PROFILE_IMAGE_URL_PATH = 'media/profile_images/'
+PROFILE_IMAGE_DEFAULT_FILENAME = 'default_profile_image'  # TODO: determine final name
```

Comment:
I think we need a comment here advising the operator that they will have to manually put the files in place when they set up the openedx instance, assuming you agree that's a reasonable setup step.  (I think trying to get the code to do something clever and automatically install files might be more complexity than it's worth.)